webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     },
     output: {
       path: path.join(__dirname, 'build'),
-      publicPath: 'http://html.eduid.docker/static/build/',
+      publicPath: 'https://dashboard.eduid.local.emergya.info/static/front-build/',
```

Comment:
I guess publicPath should be http://html.eduid.docker/static/build/ even though it doesn't work for u2f?

Reply:
Sorry I missed that, I'll fix it. I think maybe it should be
http://dashboard.eduid.docker/static/front-build/
which should work for u2f? Both the haproxy and the nginx should be configured to provide that url...